jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -114,6 +115,9 @@
 
     /** User. */
     @Nullable private String User;
+
+    /** Volumes */
+    @Nullable private Map<String, Map<?, ?>> Volumes;
```

Comment:
Everywhere else should be `AbsoluteUnixPath` for the volumes except here - this should be `String` - following how exposed ports work in terms of when and where to convert over / check and validate should be the right way to go.

Reply:
Done, changed everywhere to `AbsoluteUnixPath` except here, doing so I've found that I missed the `JsonToImageTranslator.java` which is used for pulling the base image. 

Also, when doing this, I was wondering if exposed ports and volumes from the base image should be propagated to the final image, maybe this should be another issue?  I think volumes makes more sense than ports, but maybe we should have support for both. 

What do you think @coollog

Reply:
Yep! We actually have another issue tracking the base image configuration propagation: #595 If you would like, feel free to mention this on that issue!